_posts/2024-07-10-trino-summit-2024-call-for-speakers.md

```diff
@@ -0,0 +1,80 @@
+you know that can only mean one thing: Trino Summit 2024. Much like last year,
+it will be a two-day, fully virtual event hosting a wide range of talks
+covering all things Trino on the 11th and 12th of December. Read on for more
+info, or if you're already convinced...
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-summit-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q3-EV-Trino-Fest-2024&utm_content=Blog-1">
+        Register to attend!
+    </a>
+</div>
+
+<!--more-->
+
+## Join us online
+
+Trino Summit is an event that brings together engineers, analysts, data
+scientists, and anyone else interested in using or contributing to Trino. As the
+biggest Trino event of the year, we're excited to bring together professionals
+from the big data and analytics community, so they can share experiences and
+insights, make connections, and learn from each other. The event will be
+broadcast live, and speakers will be addressing questions asked in chat, so if
+you want the full experience, make sure to register and attend while the talks
+are happening. However, if you can't make it, we'll also be posting videos of
+all talks to the Trino YouTube channel after the event, so don't fret.
+
+## Call for speakers
+
+Interested in speaking? We want to hear from everyone in the Trino community
+who has something to share. We are looking for full sessions (about 30 minutes)
+and lightning talks (10-15 minutes). We welcome beginner to highly advanced
+submissions for talks that are connected to Trino. A two-day event means we've
+got room for everything, so if you're on the fence about whether to submit a
+talk, go ahead and do it! We'll review all submissions, and we'll do our best to
+work with you to turn your talk into a smash hit. Some of the possible topics
+include:
+
+* Best practices and use cases
+* Data lake, lakehouse, and data federation architectures
+* Query federation and data migrations
+* Table formats, file formats, and metadata catalogs
+* Optimizations and performance improvements
+* Data engineering, including data cleaning, batch and streaming architectures,
+  and maintenance
+* Streaming and other data ingestion and pipelines
+* Data science workflows and analytics
+* SQL analytics, business intelligence, dashboarding and other visualizations
+* Data governance and security
+* Writing advanced SQL queries and pipelines
+* Help for Trino deployment on-premise and in the cloud
+* Developing custom connectors and other plugins
+* Contributing to Trino
+
+Want to speak?
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="https://sessionize.com/trino-summit-2024">
+        Submit a talk!
+    </a>
+</div>
+
+## Sponsor Trino Fest
+
+Starburst is the organizing sponsor of the event, but to make Trino Summit a
+smashing success, they're excited and interested in collaborating with other
```

Comment:
```suggestion
smashing success, we are excited and interested in collaborating with other
```

Reply:
I think this needs to stay "they are" because this is a Trino website blog, and the voice is Trino, not Starburst.

Reply:
fair